src/sentry/mail/activity/release.py

```diff
@@ -142,7 +155,8 @@ def get_participants(self):
                 (
                     user,
                     options.get(
-                        "org", options.get("default", UserOptionValue.committed_deploys_only)
+                        "org",
+                        options.get("default", NotificationSettingOptionValues.COMMITTED_ONLY),
```

Comment:
I'm looking into why `test_no_committers` is failing and it might be because this is filtering out users whose setting is ALWAYS for this.

Reply:
I found the bug. I was comparing enums to values. I'm working on a commit to ass a bunch of `.value`s.